lib/httputils/json.go

```diff
@@ -0,0 +1,42 @@
+package httputils
+
+import (
+	"encoding/json"
+	"net/http"
+
+	common "boscoin.io/sebak/lib/common"
+)
+
+func WriteJSON(w http.ResponseWriter, code int, v interface{}) error {
```

Comment:
Please add documentation. Also would a `MustWriteJSON` be useful ?

Reply:
If This func doesn't return an error, I will use the name `MustWriteJSON` (maybe).

Reply:
I mean, add another function, `MustWriteJSON`, which wraps this with a `panic`.

Reply:
And If someone needs to `MustWriteJSON` and make it, I am OK to make it :-)

Reply:
Ah it was just for quick & dirty tests. But I guess we can wait until the need arises.